guides/writing-cross-platform-javascript.md

```diff
@@ -0,0 +1,85 @@
+# Writing Cross-Platform JavaScript
+
+Cypress works on Linux, macOS and Windows. This includes both installing from npm, as well as for local development. Code should be written in a platform agnostic style.
+
+## Handling File Paths
+
+Throughout the code base, we access the file system in various ways, and need be concious of how we do so to ensure Cypress can be used and developed seemlessly on multiple platforms. One thing to keep in mind is file paths and file separators. macOS and Linux systems that use `/`, and Windows which uses `\`.
+
+
+As a general rule, we want to use **native paths** where possible. There are a few reasons for this. Where ever we display a file path, we want to use the native file separator, since that is what the user will expect on their platform. In general, we can use the Node.js `path` module to handle this:
```

Comment:
```suggestion

As a general rule, we want to use **native paths** where possible. There are a few reasons for this. Whereever we display a file path, we want to use the native file separator, since that is what the user will expect on their platform. In general, we can use the Node.js `path` module to handle this:
```